test/integ/pandas/test_arrow_pandas.py

```diff
@@ -12,7 +12,12 @@
 
 import pytest
 
-from snowflake.connector.options import installed_pandas, pandas
+try:
+    from snowflake.connector.options import installed_pandas, pandas  # NOQA
+except ImportError:
+    installed_pandas = False
+    pandas = None
```

Comment:
Although this test won't run for the old driver, pytest runs this part to determine if the test is runnable.